src/components/Activity/Connected.js

```diff
@@ -31,7 +31,7 @@ class Connected extends Component {
 
   onRequestCallback(result, error) {
     if (error) {
-      alert(JSON.stringify(error));
+      alert(error);
```

Comment:
Plz don't forget to unable these hundreds of alerts before the launch(or test flight)